scripts/export_tasks.py

```diff
@@ -0,0 +1,40 @@
+import ast
+import os
+import pathlib
+import json
+
+lib_to_task_map = {}
+
+
+def extract_tasks(library_name, variable_name, value):
```

Comment:
Nit, a good way to indicate this method is intended to be private

```suggestion
def _extract_tasks(library_name, variable_name, value):
```